### DISCUSSION TITLE
Review Club : Fix -norpcwhitelist, -norpcallowip, and similar corner case behavior

### DISCUSSION THREAD

**Discussion by Prabhat1308** [Bitshala/BitcoinCore-PR-Review-Club#81] [PR Reviews]
## Session Details
Date:08-08-2024
Time: IST 20:00 (UTC 14:30)
Link : https://github.com/bitcoin/bitcoin/pull/30529
Difficulty : Easy

## Notes
Make negating options the same as not setting them for ` -norpcallowip`, `-norpcbind`, `-nobind`, `-nowhitebind`, `-noexternalip`, `-noonlynet`, `-noseednode`, `-nosignetchallenge`, `-nosignetseednode`, `-notest`, and` -norpcwallet` options, instead of triggering confusing side effects and errors, like `-norpcwallet` trying to make RPC calls to the "0" wallet. This PR aims to standardise the RPCs to be consistent throughtout the codebase .

## Questions
TBD
## Learning
RPCs

**Comment by paperpsych** [comment]
@VirdiJatin Also, @Prabhat1308 I'm not sure if the tags on this one are relevant or right, let me know if they need to be changed. Also, please add 3 tags in your session details description which we can use on the thumbnail, thanks !!

![Bitshala RC (30529)](https://github.com/user-attachments/assets/516fceb5-d887-4dfc-94f0-4d39c76f49a2)
